CPAC/pipeline/engine.py

```diff
@@ -2409,9 +2410,11 @@ def ingress_pipeconfig_paths(cfg, rpool, unique_id, creds_path=None):
 
     import pandas as pd
     import pkg_resources as p
+    import sys
```

Comment:
```suggestion
    import sys

    import pandas as pd
    import pkg_resources as p
```

[pre-commit](https://github.com/FCP-INDI/C-PAC/blob/dbdacd1c6a745b4847b2c6abdf2c2f3d6150511f/CONTRIBUTING.md#pre-commit) should have caught this?

Although, if we switch from `sys.exit()` to `OrientationError()`, we don't need to `import sys` at all.